.circleci/src/workflows/release.yml

```diff
@@ -42,9 +42,6 @@ jobs:
       service: healthz
 
   - release-github-draft-trigger:
-      filters:
-        branches:
-          only: /^release.*$/
```

Comment:
this filter was redundant because the release workflow only runs on release branches